assets/training/finetune_acft_hf_nlp/environments/acpt/context/requirements.txt

```diff
@@ -5,7 +5,7 @@ mltable=={{latest-pypi-version}}
 datasets==2.12.0
 mpi4py==3.1.4
 sentencepiece==0.1.99
-transformers==4.33.2
+transformers==4.33.4
```

Comment:
Should be 4.34.0 https://pypi.org/project/transformers/#history

Reply:
fixed it